cilium/ebpf.io-summit
# Add videos and slides

**Describe what changes this pull request brings**
This PR removes the schedule and adds a section with links to videos and slides for talks.

**Steps to test**
1. Open [the page](https://deploy-preview-53--ebpf-summit-2021.netlify.app/summit-2022/)
2. Confirm that everything looks and works as expected

<details> 
<summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/68022519/199235644-c918adf6-a507-41c9-b590-51f54229ff05.png) 
</details> 



**References**

1. [Design](https://www.figma.com/file/L49LK6NTH0A5PVwGSW25ub/EBPF-Summit?node-id=2252%3A4664)
